syndicate/connection/lambda_connection.py

```diff
@@ -498,14 +501,19 @@ def update_code_source(self, lambda_name, s3_bucket, s3_key,
                                          Publish=publish_version)
 
     def update_event_source(self, uuid, function_name, batch_size,
-                            batch_window=None, filters: Optional[List] = None):
+                            batch_window=None, filters: Optional[List] = None,
+                            enable_batch_report_failures=None):
         params = dict(
             UUID=uuid, FunctionName=function_name, BatchSize=batch_size
         )
         if batch_window is not None:
             params['MaximumBatchingWindowInSeconds'] = batch_window
         if filters is not None:
             params['FilterCriteria'] = {'Filters': filters}
+        if enable_batch_report_failures is not None and enable_batch_report_failures:
```

Comment:
Please stick with lines of code no longer than 80 symbols
This line of code will work fine and be more Pythonic just like that
"if enable_batch_report_failures:"

Reply:
I optimized this if statement, and another one(which is the same, but in different function)